controls/mysql_conf.rb

```diff
@@ -45,6 +45,7 @@
   mysql_config_file = "#{mysql_config_path}my.cnf"
   mysql_log_group = 'adm'
   service_name = 'mysql'
+  service_name = 'mariadbd' if os[:release] >= '11'
```

Comment:
I think this will generate problems on Ubuntu

Reply:
Yes, that shouldn't be part of this PR, I added it by mistake.